execution_engine/src/core/engine_state/mod.rs

```diff
@@ -737,6 +740,19 @@ where
             tracking_copy.borrow_mut().write(validator_slots_key, value);
         }
 
+        if let Some(new_auction_delay) = upgrade_config.new_auction_delay() {
```

Comment:
Update the spec.

Reply:
Spec?

Reply:
https://casperlabs.atlassian.net/wiki/spaces/EN/pages/123437404/Specifications

There's one for genesis and one for system upgrades.

Reply:
Spec for system upgrades has been updated